validation/userRegisterValidator.ts

```diff
@@ -0,0 +1,12 @@
+import * as yup from "yup"; 
```

Comment:
I recommend that this file be renamed to *registrationValidator* to simplify it.